contrib/opencensus-ext-pymongo/opencensus/ext/pymongo/trace.py

```diff
@@ -16,25 +16,26 @@
 
 from pymongo import monitoring
 
+from google.rpc import code_pb2
+
 from opencensus.trace import execution_context
 from opencensus.trace import span as span_module
 
 
 log = logging.getLogger(__name__)
 
-MODULE_NAME = 'pymongo'
+MODULE_NAME = "pymongo"
 
-COMMAND_ATTRIBUTES = ['filter', 'sort', 'skip', 'limit', 'pipeline']
+COMMAND_ATTRIBUTES = ["filter", "sort", "skip", "limit", "pipeline"]
```

Comment:
Curious, is there a reason to change single quote to double quote?
If it is for consistency, I can still see lots of single quotes at line57, etc.

Reply:
Missed that, sorry.

Reply:
PS: shouldn't the linter get this?